pages/release-notes-archive.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: page
-title: Fluo Release Notes Archive
+title: Release Notes Archive
 permalink: "/release-notes/"
 ---
 
-Below are the release notes for all Fluo releases:
+Below are the release notes for all Apache Fluo releases:
```

Comment:
Currently there are not Apache Fluo release to have release notes for ... so this is misleading....